index.html

```diff
@@ -5,9 +5,173 @@
     <link rel="stylesheet" href="/node_modules/normalize.css/normalize.css">
     <link rel="stylesheet" href="styles/styles.css">
-    <title>Base Project</title>
+    <link href="https://fonts.googleapis.com/css?family=Anaheim|Berkshire+Swash&display=swap" rel="stylesheet">
+    <title>Restaurant Website</title>
 </head>
 <body>
-    
+
+    <section class="header">
+        <div class="top-container">
+        <div class="logo"> <img src="/images/logo.png" alt="Lambda logo"> </div>
+        <nav class="topnav">
+            <ul>
+                <li><a href="#home" class="current">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#ingredients">Ingredients</a> </li>
+                <li><a href="#menu">Menu</a></li>
+                <li> <a href="#reviews">Reviews</a></li>
+                <li><a href="#reservations">Reservations</a></li>
+            </ul>
+        </nav>
+        </div>
+
+        <h1> 
+            the right ingredients <br> for the right food
+        </h1>
+        <p>
+            <img src="/images/divider-white.png" alt="White divider">
+        </p>
+        <div class="buttons"> 
+            <button class="btn-1">BOOK A TABLE</button>
+            <button class="btn-2">SEE THE MENU</button>
+        </div>
+    </section>
+
+    <div class="cook">
+        <div> 
+            <h2>Just the right food</h2> 
+            <img src="/images/divider-gold.png" alt="Golden divider">
+            <p>If you've been to one of our restaurants, you've seen - and tasted - what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lamba cakes, muffins, and gourmet coffes make us hard to resist! Stop in today and check us out!</p>
+            <img id="about-cook" src="/images/about-cook.png" alt="Cook decorating a plate">
+        </div>
+        <div>
+            <img id="steak-plate" src="/images/about-dish.png" alt="Plate">
+        </div>
+    </div>
+
+    <div class="ingredients">
+        <div class="container">
+            <h2>Fine ingredients</h2>
+            <img src="/images/divider-white.png" alt="White divider">
+            <p>If you've been to one of our restaurants, you've seen - and tasted - what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lamba cakes, muffins, and gourmet coffes make us hard to resist! Stop in today and check us out!</p>
+            <div class="small-container">
+                <img class="miniatures" src="/images/ingredients-grains.png" alt="Grains">
+                <img class="miniatures" src="/images/ingredients-spices.png" alt="Spices">
+                <img class="miniatures" src="/images/ingredients-bread.png" alt="Bread">
+            </div>
+        </div>
+    </div>
+
+    <div class="menu">
+        <div class="item">
+            <h2>Appetisers <br><img src="/images/divider-gold.png" alt="Golden divider"></h2> 
```

Comment:
On the subject of the divider, there are a myriad number of ways to handle this, but I wouldn't place an `<img>` inside a `<h2>` tag. Actually it's not valid HTML to do that.

I would place the image inside it's own element, and simply place it underneath. You could use a `<div>`, or even better - use the `<hr>` tag (we haven't covered this). `<hr>` means horizontal rule. Literally a dividing line. By default it looks just like that - a dividing line, but you can restyle it using CSS. For  example the `divider-gold.png` image could be used as a background image for the `<hr>` tag. This website explains it: https://www.formget.com/css-hr/